packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/components/list/OvirtCredentialsList.tsx

```diff
@@ -39,9 +39,10 @@ export const OvirtCredentialsList: React.FC<ListComponentProps> = ({ secret, rev
       description: t("If true, the provider's CA certificate won't be validated."),
       helperTextPopover: (
         <Trans t={t} ns="plugin__forklift-console-plugin">
-          {
-            "Note: If 'Skip certificate validation' is selected, migrations from this provider will not be secure.<br><br> Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed."
-          }
+          Note: If {'<strong>'}Skip certificate validation{'</strong>'} is selected, migrations from
+          this provider will not be secure.{'<br><br>'} Insecure migration means that the
+          transferred data is sent over an insecure connection and potentially sensitive data could
+          be exposed.
```

Comment:
```suggestion
          Note: If <strong>Skip certificate validation</strong> is selected, migrations from
          this provider will not be secure.<br />
          <br />
          Insecure migration means that the
          transferred data is sent over an insecure connection and potentially sensitive data could
          be exposed.
```

* not just here, please change in all places you use strings instead of components